pkg/crc/cluster/cluster.go

```diff
@@ -104,6 +105,29 @@ func GetRAMUsage(sshRunner *ssh.Runner) (int64, int64, error) {
 	return ramSize, ramUsage, nil
 }
 
+func GetCPUUsage(sshRunner *ssh.Runner) ([]int64, error) {
+	cmd := "top -n1 -1 -b -w 79  | grep %Cpu | awk '{print $3}'"
+	out, _, err := sshRunner.Run(cmd)
```

Comment:
I was a bit worried that the output would be locale-dependant, but this seems ok, if I set `LC_ALL`/`LANG`, manually start the daemon, `sshRunner.Run("printenv")` does not inherit the locale changes, which is good (floats in French locale are "6,2", not "6.2")